.ruby-version

```diff
@@ -1 +0,0 @@
-2.1.1
```

Comment:
Because there's no point enforcing a ruby version at all. It was initially deleted because fuck no was I going to wait until ruby 2.1.1 was installed, and I just left it in after talking to people and basically getting a consensus that .ruby-version is pointless in public projects.

If you really want, I can add another commit which reintroduces the .ruby-version.